### DISCUSSION TITLE
Create orders from the admin

### DISCUSSION THREAD

**Discussion by jalfargentina** [medusajs/medusa#5778] [Feature Requests]
As a developer and owner of an ecommerce it would be very useful to be able to create orders from the same admin panel, without the need to create a storefront.

In my case it would be very useful to manage everything from the admin panel because my customers make many orders from whatsapp and the role of medusa js in this case would be to be a dashboard to check stock, upload products and manage orders from one place for the entire sales team.

Thanks!!

**Comment by srindom** [comment]
Does the Draft Order functionality cover your needs or are you looking for further capabilities?

**Reply by mahimsafa** [reply]
How to draft the order? I dont see any optio there.

![image](https://github.com/user-attachments/assets/5d05af54-1132-4095-8c7b-0514d9f6277d)

**Comment by maclive** [comment]
yes draft order
